ginlong_solis.yaml

```diff
@@ -216,7 +218,7 @@ pred_bat:
   # Or manually set it to the correct sensor names e.g:
   # sensor.octopus_energy_electricity_xxxxxxxxxx_xxxxxxxxxxxxx_current_rate
   # sensor.octopus_energy_electricity_xxxxxxxxxx_xxxxxxxxxxxxx_export_current_rate
-  metric_octopus_import: 're:(sensor.(octopus_energy_|)electricity_[0-9a-z]+_[0-9a-z]+_current_rate)'
+  metric_octopus_import: sensor.octopus_energy_electricity_19m1337498_1610032016836_current_rate
```

Comment:
This looks like a typo to hard-write your meter number?

Reply:
Correct! I was having issues with Octopus last night. Will edit this out.